README.md

```diff
@@ -1,5 +1,9 @@
 # kuberpult Readme for users
 
+## Etymology
+
+Kuberpult is a catapult for [kubernetes](https://kubernetes.io/) :) it catapults the kubernetes containers to different stages (like production).
```

Comment:
Technically not _kubernetes_ containers. Maybe rather "catapults the application containers into different stages' kubernetes clusters"

Reply:
I rephrased it.